### PR TITLE
resource/alicloud_ess_scaling_rule: Add attributes of scale_out_evaluation_count,scale_in_evaluation_count and min_adjustment_magnitude

### PR DESCRIPTION
**Pull request by fuliu-zln**
…ation_count,scale_in_evaluation_count and min_adjustment_magnitude

### CODE REVIEW

File: alicloud/resource_alicloud_ess_scaling_rule.go [hunk 1]
```diff
@@ -66,6 +66,29 @@ func resourceAlicloudEssScalingRule() *schema.Resource {
...
 			},
+			"scale_in_evaluation_count": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+			"scale_out_evaluation_count": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_ess_scaling_rule.go:77]: Computed 要checkset

Comment (fuliu-zln) [reply alicloud/resource_alicloud_ess_scaling_rule.go:77]: done

File: alicloud/resource_alicloud_ess_scaling_rule_test.go [hunk 2]
```diff
@@ -471,7 +471,266 @@ func TestAccAliCloudEssScalingRuleMulti(t *testing.T) {
...
+					"metric_name":               "CpuUtilization",
+					"target_value":              "20.1",
+					"estimated_instance_warmup": "200",
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(nil),
+				),
+			},
+			{
+				ResourceName:      resourceId,
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_ess_scaling_rule_test.go:681]: 这个Import放到最后一步

Comment (fuliu-zln) [reply alicloud/resource_alicloud_ess_scaling_rule_test.go]: done

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_ess_scaling_rule.go [hunk 3]
```diff
@@ -241,28 +266,25 @@ func resourceAliyunEssScalingRuleUpdate(d *schema.ResourceData, meta interface{}
...
+		}
 		if d.HasChange("adjustment_value") {
 			request.AdjustmentValue = requests.NewInteger(d.Get("adjustment_value").(int))
 		}
 		if d.HasChange("cooldown") {
 			request.Cooldown = requests.NewInteger(d.Get("cooldown").(int))
 		}
 	case string(TargetTrackingScalingRule):
-		//if d.HasChange("alarm_dimension") {
-		//	steps := make([]ess.ModifyScalingRuleAlarmDimension, 0)
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_ess_scaling_rule.go:252]: 这里为啥去掉了

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**